core/crypto/ecdsa_test.go

```diff
@@ -57,6 +57,8 @@ func TestVerify(t *testing.T) {
 	}
 }
 
+var benchVerifyR bool
```

Comment:
Why R? I'm curious about its origin - perhaps it's time for a creative renaming! Any suggestions for what R could stand for? (Wrong answers only, of course!)

Reply:
R is for Result.